docs/usage/mnist.rst

```diff
@@ -6,9 +6,10 @@ model for prediction via the DFFML CLI and HTTP API.
 
 Download the files and verify them with ``sha384sum``.
 
+.. literalinclude:: /../examples/MNIST/image_data.sh
+
 .. code-block:: console
 
-    $ curl -sSLO "http://yann.lecun.com/exdb/mnist/{train-images-idx3,train-labels-idx1,t10k-images-idx3,t10k-labels-idx1}-ubyte.gz"
```

Comment:
Did you mean to remove this line?

Reply:
This `curl` command and the `sha384sum` command should go to `image_data.sh`

Reply:
Oops! I was testing something, must've forgot to change it back. Sorry!